.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Checklist
+- Code is compiling ✅ ❌
+- Code is prettified ✅ ❌
+- Contains unit tests ✅ ❌
+- Contains breaking changes ✅ ❌
+
+#### Web specific
+- Contains e2e tests ✅ ❌
+- Is accessible ✅ ❌
+- Compatible with Studio ✅ ❌
+- Cross-browser compatible ✅ ❌
+
+#### Native specific
+- Works in Android ✅ ❌
+- Works in iOS ✅ ❌
+- Works in Tablet ✅ ❌
+
+#### Feature specific
+- Comply with designs ✅ ❌
+- Comply with PM's requirements ✅ ❌
+
+**_Please remove unnecessary emojis and sections and this comment before proceed_**
+
+## This PR contains
+- [ ] Bug fix
+- [ ] Feature
+- [ ] Refactor
+- [ ] Documentation
+- [ ] Other (describe)
+
+## What is the purpose of this PR?
+_Please add your explanation about the purpose of the PR_
```

Comment:
IMO we can have "..." here because the section name describes what the content should be.

Reply:
Agreed